main-site/components/faq-section/faq/Faq.styles.ts

```diff
@@ -0,0 +1,20 @@
+import styled from 'styled-components';
+import { max } from '../../../../shared-ui/lib/responsive';
+import { colors } from '../../../../shared-ui/style/colors';
+
+export const StyledFaqContainer = styled.div`
+  position: relative;
+  width: 60%;
+  margin: 0 auto;
+  color: ${colors.TEXT_BROWN}
+  position: relative;
+  padding: 4em;
+  text-align: left;
+  padding-bottom: 0;
+
+@media ${max.tablet} {
+    width: 75%;
```

Comment:
Can we change this to 80%? It looks slightly too narrow on mobile compared to the design